I am seeing the following errors in tests:
```py
___________________ ERROR collecting test_build_from_docs.py ___________________
ImportError while importing test module '$PREFIX/lib/python3.9/site-packages/bluesky_live/tests/test_build_from_docs.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/bluesky_live/tests/test_build_from_docs.py:1: in <module>
    from ..run_builder import (
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/bluesky_live/run_builder.py:7: in <module>
    from .bluesky_run import DocumentCache, BlueskyRun
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/bluesky_live/bluesky_run.py:9: in <module>
    from .document import Start, Stop, Descriptor
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/bluesky_live/document.py:4: in <module>
    from dask.base import normalize_token
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/dask/base.py:15: in <module>
    from tlz import merge, groupby, curry, identity
E   ModuleNotFoundError: No module named 'tlz'
_____________________ ERROR collecting test_run_builder.py _____________________
ImportError while importing test module '$PREFIX/lib/python3.9/site-packages/bluesky_live/tests/test_run_builder.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/bluesky_live/tests/test_run_builder.py:6: in <module>
    from ..run_builder import RunBuilder
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/bluesky_live/run_builder.py:7: in <module>
    from .bluesky_run import DocumentCache, BlueskyRun
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/bluesky_live/bluesky_run.py:9: in <module>
    from .document import Start, Stop, Descriptor
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/bluesky_live/document.py:4: in <module>
    from dask.base import normalize_token
../_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python3.9/site-packages/dask/base.py:15: in <module>
    from tlz import merge, groupby, curry, identity
E   ModuleNotFoundError: No module named 'tlz'
=========================== short test summary info ============================
ERROR test_bluesky_run.py
ERROR test_build_from_docs.py
ERROR test_run_builder.py
!!!!!!!!!!!!!!!!!!! Interrupted: 3 errors during collection !!!!!!!!!!!!!!!!!!!!
============================== 3 errors in 0.74s ===============================
```

@danielballan, any pointers?

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -24,11 +24,17 @@ requirements:
     - dask-core >=2
     - entrypoints
     - event-model >=1.16.0
-    - fsspec
     - numpy
     - pandas
-    - toolz
     - xarray
+    # TODO: remove these dependencies once they are fixed in
+    # https://anaconda.org/anaconda/dask-core/files.  See
+    # https://github.com/conda-forge/dask-core-feedstock/pull/93/files for more
+    # details.
+    - cloudpickle >=1.1.1
+    - fsspec >=0.6.0
+    - partd >=0.3.10
+    - toolz >=0.8.2
 
 test:
   requires:

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -24,6 +24,7 @@ requirements:
     - dask-core >=2
     - entrypoints
     - event-model >=1.16.0
+    - fsspec
     - numpy
     - pandas
     - toolz

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -26,6 +26,7 @@ requirements:
     - event-model >=1.16.0
     - numpy
     - pandas
+    - toolz
     - xarray
 
 test:

